### DISCUSSION TITLE
Do you consider incorporating open-source search solutions, such as enabling the network search button

### DISCUSSION THREAD

**Discussion by windkwbs** [LostRuins/koboldcpp#923] [Ideas]
Do you consider incorporating open-source search solutions, such as enabling the network search button

**Comment by LostRuins** [comment]
I would like to, but right now there are no public search APIs available for free that allow cross origin browser access.

**Reply by windkwbs** [reply]
The open source project "open webui" has several open source network search solutions that can be considered for reference,You can refer to them
